jira_integration/jira_integration.php

```diff
@@ -0,0 +1,111 @@
+<?php
+// TODO: this header should not be required...
+header('Content-Type: text/plain; charset=UTF-8');
+
+// Only watch the dev environment
+if ($_ENV['PANTHEON_ENVIRONMENT'] != 'dev') {
```

Comment:
Indeed, checking to see if a commit exists on other branches is [complicated, and not guaranteed to work](http://stackoverflow.com/questions/16304574/how-to-list-branches-that-contain-an-equivalent-commit). Exact commits [can be easily detected](http://stackoverflow.com/questions/1419623/how-to-list-branches-that-contain-a-given-commit), but this misses those that are moved via cherry-pick, et. al.

Not sure how far we should go here.